services/postprocessing/pkg/config/config.go

```diff
@@ -23,15 +23,15 @@ type Config struct {
 // Postprocessing definces the config options for the postprocessing service.
 type Postprocessing struct {
 	Events          Events        `yaml:"events"`
-	Virusscan       bool          `yaml:"virusscan" env:"POSTPROCESSING_VIRUSSCAN" desc:"should the system do a virusscan? Needs antivirus service"`
-	Delayprocessing time.Duration `yaml:"delayprocessing" env:"POSTPROCESSING_DELAY" desc:"the sytem sleeps for this time while postprocessing"`
+	Virusscan       bool          `yaml:"virusscan" env:"POSTPROCESSING_VIRUSSCAN" desc:"Needs as prerequisite the antivirus service to be enabled and configured. should the system do a virusscan?"`
```

Comment:
This is actually correct. Additionally to the antivirus service being running this envvar needs to be set to `true` to enable virusscanning during postprocessing.

An alternative would be to have this set to `false` and trigger virus scans only manually or on download.

Reply:
In a nutshell, we are doing something afterwards (postprocessin) while we are doing it right now (upload). Absolutely not confusing and totally clear 🤣

Reply:
True. The word `upload` is confusing here. Better with `postprocessing`?

Reply:
Lets discuss this with a call after Christmas. I dont have a brain for this now.